feconf.py

```diff
@@ -114,10 +114,10 @@
 MAX_LEARNER_PLAYLIST_ACTIVITY_COUNT = 10
 
 # The minimum number of training samples required for training a classifier.
-MIN_TOTAL_TRAINING_EXAMPLES = 50
+MIN_TOTAL_TRAINING_EXAMPLES = 5
```

Comment:
@prasanna08 So, to test handlers, we need to load demo explorations. Only one demo exploration has trainable states and it has like 10 training examples. The previous number of min_total_examples was just some random number we put up. When this actually gets into prod, we will decide on a concrete value for this.